composer.json

```diff
@@ -28,5 +28,10 @@
         "psr-4": {
             "AppTest\\": "test/AppTest/"
         }
+    },
+    "scripts": {
+        "phpcs": "phpcs",
+        "server": "php -S localhost:8080 -t public",
```

Comment:
Yeah, I've seen that before. I won't use this functionality, I just put it in just in case. Shall we take it out?